packages/shared/src/components/PostOptionsMenu.tsx

```diff
@@ -426,23 +434,34 @@ export default function PostOptionsMenu({
     });
   }
 
+  const isMobile = useViewSize(ViewSize.MobileL);
```

Comment:
I feel we are going to need this logic quite often right?
Should we simply wrap it into it's own abstraction?

Maybe even PortalMenu? (but not sure if other occasions use it)
Otherwise it could be `ContextMenu` maybe? 
inside it will evalaute whether to show drawer or portal

WDYT?

Reply:
Yeah, I was hoping to do that too, but it is a bit more complicated with the PortalMenu that we have than the dropdown. Though we have a separate ticket for that, to migrate all context menus. We can maybe do that there. Wdyt?

Reply:
We just wanted to push this first as it is a bit higher priority than the others.

Reply:
Ok, maybe note this comment down on that ticket so we have reference.
Happy to not block, but eventually would love one generic component for this 🙏

Reply:
Yeah, just our context menus have different implementations, old ones not using the generic component, so I didn't want to spend the time for now. I will add a comment here.

Reply:
Done 🥳  https://dailydotdev.atlassian.net/browse/MI-128?focusedCommentId=11464